starter-code/TicTacToe/app/src/main/java/ly/generalassemb/drewmahrt/tictactoe/GameActivity.java

```diff
@@ -1,13 +1,213 @@
 package ly.generalassemb.drewmahrt.tictactoe;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.Toast;
 
 public class GameActivity extends AppCompatActivity {
+    private TextView gameMessage;
+    private TextView box1, box2, box3, box4, box5, box6, box7, box8, box9;
+    private RelativeLayout gameScreen;
+    private Button resetBoard;
+    private GameBoard game;
+    int turnNumber = 0;
+    private Button back;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_game);
+
+        Intent intent = getIntent();
+
+        resetBoard = (Button)findViewById(R.id.clear_board);
+        back = (Button)findViewById(R.id.back);
+
+        box1 = (TextView)findViewById(R.id.textView);
+        box2 = (TextView)findViewById(R.id.textView2);
+        box3 = (TextView)findViewById(R.id.textView3);
+        box4 = (TextView)findViewById(R.id.textView4);
+        box5 = (TextView)findViewById(R.id.textView5);
+        box6 = (TextView)findViewById(R.id.textView6);
+        box7 = (TextView)findViewById(R.id.textView7);
+        box8 = (TextView)findViewById(R.id.textView8);
+        box9 = (TextView)findViewById(R.id.textView9);
+
+        gameScreen = (RelativeLayout)findViewById(R.id.game_screen);
+
+        gameMessage = (TextView)findViewById(R.id.game_message_text);
+        final String playerName1 = intent.getStringExtra("PlayerOne");
+        final String playerName2 = intent.getStringExtra("PlayerTwo");
+        game = new GameBoard(playerName1, playerName2);
+        gameMessage.setText(playerName1+"'s Turn");
+
+        View.OnClickListener gameClick = new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if(turnNumber == 10) {
+                    gameMessage.setText("Reset to play again");
+                }
+                if(turnNumber == 9) {
+                    gameMessage.setText("It's a tie!");
+                    turnNumber++;
+                } else {
+                    switch(v.getId()) {
+                        case R.id.textView:
+                            if (box1.getText().equals(" ")) {
+                                game.addMove(0, 0, box1);
+                                if (game.isGameOver()) {
+                                    gameMessage.setText(game.currentPlayer() + " Won!");
+                                    turnNumber = 10;
+                                    game.stopGame();
+                                } else {
+                                    turnNumber++;
+                                }
```

Comment:
You're repeating this logic a bunch of times - I suggest moving it to a helper method and passing in just the values that are different depending on which TextView was clicked